x-pack/legacy/plugins/siem/public/components/links/index.tsx

```diff
@@ -254,27 +253,29 @@ const ReputationLinkComponent: React.FC<{
               ))}
           </EuiFlexGroup>
         </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <DefaultFieldRendererOverflow
-            rowItems={ipReputationLinks}
-            idPrefix="moreReputationLink"
-            render={rowItem => {
-              return (
-                isReputationLink(rowItem) && (
-                  <ExternalLink
-                    url={rowItem.url_template}
-                    overflowIndexStart={overflowIndexStart}
-                    allItemsLimit={allItemsLimit}
-                  >
-                    <>{rowItem.name ?? domain}</>
-                  </ExternalLink>
-                )
-              );
-            }}
-            moreMaxHeight={DEFAULT_MORE_MAX_HEIGHT}
-            overflowIndexStart={overflowIndexStart}
-          />
-        </EuiFlexItem>
+        {ipReputationLinks.length && (
```

Comment:
I shouldn't put `ipReputationLinks.length` like this, it actually prints the number